components/organisms/LessonsWithinACourseAdminPortal/LessonsWithinACourseAdminPortal.tsx

```diff
@@ -30,7 +32,22 @@ export const LessonsWithinACourseAdminPortal = () => {
     ? [selectedLesson._id]
     : [];
 
-  const handleEditClick = (row: any) => {};
+  const handleEditClick = (row: any) => {
+    if (row) {
+      dispatch(resourceEditAdmin(row));
+      router.push(
+        "/dashboard/adminPanel/courses/courseTopicManage/lessonsWithinACourse/editLesson"
+      );
+    } else {
+      Swal.fire({
+        title: "Error al seleccionar el recurso",
+        text: "Hablar con el equipo de desarrollo",
```

Comment:
I think this line should be more friendly... something like:

```suggestion
        text: "Ponte en contacto con el equipo de desarrollo",
```

Reply:
![image](https://github.com/user-attachments/assets/ef5a6fd1-c063-45cc-ac60-e1cd33b9af5b)